documentation/how-to/api-discovery.md

```diff
@@ -109,58 +99,72 @@ Here is the schema with example data, omitting some optional fields:
     "record_publish_date": "2021-05-27"
     }
     ```
+ | Field     | Description                              |
+ | --------------- | ---------------------------------------------------------------- |
+ | `kind` |     |
+ | `name`      | Unique dataset name, not displayed                  |
+ | `title`      | API title shown in the API Directory                 |
+ |`notes`      | API description, supports Markdown formatting                |
+ |`tags`      | Keywords, may be used for search in the future                 |
+ | `organization`      | Ministry or agency associated with the API ([options](<https://api.gov.bc.ca/ds/api/v2/organizations>)) that must match the organization associated with the Gateway                |
+ | `organizationUnit`      | Organization business sub-unit ([options](https://api.gov.bc.ca/ds/api/v2/organizations/<organization>)) |
```

Comment:
```suggestion
 | `organizationUnit`      | Organization business sub-unit. See `https://api.gov.bc.ca/ds/api/v2/organizations/<organization>` for options.
```
This URL includes a placeholder which the user needs to modify, so can't include an actual link.